### PR TITLE
shaarli_client: add function to delete a link

### PR DESCRIPTION
**Pull request by dkrm0**

### CODE REVIEW

File: shaarli_client/client/v1.py [hunk 1]
```diff
@@ -304,3 +314,7 @@ def delete_tag(self, resource, params):
         """Delete a tag"""
         self._check_endpoint_params('delete-tag', params)
         return self._request('DELETE', 'tags/%s' % resource, params)
+
+    def delete_link(self, resource):
+        """Delete a link"""
+        return self._request('DELETE', 'links/%d' % resource, {})
```
Comment (nodiscc) [inline shaarli_client/client/v1.py:320]: you need to add `, verify_certs` as the last argument to `_request()` - https://github.com/shaarli/python-shaarli-client/blob/master/shaarli_client/client/v1.py#L249

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: shaarli_client/client/v1.py (hunk 1 above)
Comment (dkrm0) [reply shaarli_client/client/v1.py:320]: None of the others functions have the `verify_certs` argument

Comment (nodiscc) [reply shaarli_client/client/v1.py:320]: I will recheck everything, `verify_certs` was added in January but tests have been failing for longer than that (https://github.com/shaarli/python-shaarli-client/pull/46). It's possible I missed someting

**Comment by nodiscc** [comment]
Rebased/moved to https://github.com/shaarli/python-shaarli-client/pull/56

**Pull request closed without merging**